__tests__/app.test.js

```diff
@@ -349,7 +349,28 @@ describe('/api/comments/:comments_id', () => {
             .expect(400)
             .then(({ body: { msg } }) => {
                 expect(msg).toBe("Bad request.");
-            })
+            });
         });
     });
 });
+
+describe('/api/users', () => {
+    describe('GET', () => {
+        test('GET:200 responds with an array of user objects, each with a "username", "name" and "avatar_url" property', () => {
+            return request(app)
+            .get('/api/users')
+            .expect(200)
+            .then(({ body: { users } }) => {
+                expect(users.length).toBe(4);
+                users.forEach((user) => {
+                    expect(user).toHaveProperty('username');
```

Comment:
If you wanted to you could neaten this up with the jest matcher expect object containing. You're then asserting both the key and value data type in one.